### DISCUSSION TITLE
Welcome to gost Discussions!

### DISCUSSION THREAD

**Discussion by ginuerzh** [go-gost/gost#353] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by cnwhy** [comment]
感谢作者做出这如此好用的工具；工作，生活上都能用上，真是太赞了；
